Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonThemeComboBox.cs

```diff
@@ -0,0 +1,29 @@
+﻿#region BSD License
+/*
+ * 
+ * Original BSD 3-Clause License (https://github.com/ComponentFactory/Krypton/blob/master/LICENSE)
+ *  © Component Factory Pty Ltd, 2006 - 2016, (Version 4.5.0.0) All rights reserved.
+ * 
+ *  New BSD 3-Clause License (https://github.com/Krypton-Suite/Standard-Toolkit/blob/master/LICENSE)
+ *  Modifications by Peter Wagner(aka Wagnerp) & Simon Coghlan(aka Smurf-IV), et al. 2017 - 2021. All rights reserved. 
+ *  
+ */
+#endregion
+
+namespace Krypton.Toolkit
+{
+    public class KryptonThemeComboBox : KryptonComboBox
+    {
+        #region Constructor
+        /// <summary>Initializes a new instance of the <see cref="KryptonThemeComboBox" /> class.</summary>
+        public KryptonThemeComboBox()
+        {
+            DropDownStyle = ComboBoxStyle.DropDownList;
+
+            ThemeManager.PropagateThemeSelector(this);
+
+            SelectedIndex = 24;
```

Comment:
Would this be better as selected text, so that if new are added then "we" do not have to remember to alter this.
No, scrap that.
It should be reading the parent `Krypton manager` and setting that as the default.

Reply:
i.e. setting the text to the current theme name?

Reply:
Yes, and also protecting the `On index changed` to not fire during that time.